src/maheshKJun20/NumbersExample/FibonacciNumbers.java

```diff
@@ -0,0 +1,24 @@
+package maheshKJun20.NumbersExample;
+
+public class FibonacciNumbers {
+	
+	void displayFibonacciNumber(int num) {
+		int firstValue = 1, secondValue = 1, newValue = 0;
+		System.out.print("Fibonacci Series for " + num + " Output: 1 1");
+		for (int index=1;index<num;index++) {
+			newValue = firstValue + secondValue;
+			firstValue = secondValue;
+			secondValue = newValue;
+			System.out.print(" " + newValue);
+		}
+		System.out.println("");
```

Comment:
Rather printing extra statement add \n on line 7 and you'll get newline when you'll call method for next time.
System.out.print("\nFibonacci Series for " + num + " Output: 1 1");

Reply:
Updated as per above suggestion. Thanks for suggestion